source/common/quic/envoy_quic_stream.h

```diff
@@ -25,7 +25,7 @@ class EnvoyQuicStream : public virtual Http::StreamEncoder,
 public:
   // |buffer_limit| is the high watermark of the stream send buffer, and the low
   // watermark will be half of it.
-  EnvoyQuicStream(uint32_t buffer_limit, QuicFilterManagerConnectionImpl& filter_manager_connection,
+  EnvoyQuicStream(uint64_t buffer_limit, QuicFilterManagerConnectionImpl& filter_manager_connection,
```

Comment:
Do we really need this change to fix updateBytesBuffered()? I think in some Envoy interface the buffer_limit is returned as uint32_t.

Reply:
No they aren't necessary for this issue.
I've removed them so this change is cleaner.